lalrpop/src/build/mod.rs

```diff
@@ -348,8 +348,14 @@ fn emit_recursive_ascent(session: &Session, grammar: &r::Grammar) -> io::Result<
                                                         "super",
                                                         &mut rust)),
 
-            r::LrCodeGeneration::TestAll =>
-                try!(lr1::codegen::test_all::compile(&grammar, user_nt, start_nt, &states, &mut rust)),
+            r::LrCodeGeneration::TestAll => {
+                try!(lr1::codegen::test_all::compile(&grammar, user_nt, start_nt, &states, &mut rust));
+                rust!(rust,
```

Comment:
oh, is this just a hack b/c it doesn't work with `ascent` yet? 

maybe we can just have a test here that says like "FIXME() -- if the grammar uses error, just generate the parse-table variant for now"